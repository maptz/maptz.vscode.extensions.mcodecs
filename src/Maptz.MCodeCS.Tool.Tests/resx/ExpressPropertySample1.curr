using System;
using System.Threading.Tasks;

namespace MyNamespace{

	public class MyClass
	{

		public MyClass(IInfo info)
		{
			{{CURSOR}}"new S";
			this.Info = info;
		}

		public IInfo Info { get; set; }

		public Task DoSometihngAsync()
		{
			
		}
	}
}