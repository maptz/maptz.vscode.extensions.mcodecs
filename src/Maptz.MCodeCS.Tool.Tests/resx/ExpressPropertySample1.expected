[{"Kind":4,"Start":69,"End":267,"NewText":"\r\n{\r\n    public class MyClass\r\n    {\r\n        private String String\r\n        {\r\n            get;\r\n            set;\r\n        }\r\n\r\n        public MyClass(IInfo info)\r\n        {\r\n            this.String = \"new S\";\r\n            this.Info = info;\r\n        }\r\n\r\n        public IInfo Info\r\n        {\r\n            get;\r\n            set;\r\n        }\r\n\r\n        public Task DoSometihngAsync()\r\n        {\r\n        }\r\n    ","FilePath":"X:\\+++DEV\\MaptzGitHub\\vscode\\maptz.vscode.extensions.mcodecs\\src\\Maptz.MCodeCS.Tool.Tests\\bin\\Debug\\netcoreapp2.1\\SomeFile.cs"}]